process/factory/interceptorscontainer/shardInterceptorsContainerFactory_test.go

```diff
@@ -477,7 +481,7 @@ func TestShardInterceptorsContainerFactory_With4ShardsShouldWork(t *testing.T) {
 	numInterceptorsUnsignedTxs := numInterceptorTxs
 	numInterceptorsRewardTxs := numInterceptorTxs
 	numInterceptorHeaders := 1
-	numInterceptorMiniBlocks := noOfShards + 1
+	numInterceptorMiniBlocks := noOfShards + 1 + allShardNumber
```

Comment:
+2 to comply with the rest of the test files

Reply:
fixed